lib/device-settings.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import log from './logger';
+
+class DeviceSettings {
+
+  constructor() {
+    this._settings = {};
+    // this is where default settings can be declared
+    this._settings.ignoreUnimportantViews = false;
```

Comment:
we should probably expose defaults as an argument to the constructor so that each driver can define them. basedriver shouldn't know about android views